omnibus/config/software/postgresql133-bin.rb

```diff
@@ -0,0 +1,38 @@
+# limitations under the License.
+#
+
+name "postgresql133-bin"
```

Comment:
Looking at: https://www.postgresql.org/support/versioning/ we should not need to ship a binary everytime we upgrade from 13.x to 13.y